Wake on LAN: migrate action to a dedicated action page (#46070)
home-assistant/home-assistant.github.io: source/_actions/wake_on_lan.send_magic_packet.markdown

---
title: "Send magic packet"
action: wake_on_lan.send_magic_packet
domain: wake_on_lan
description: "Sends a magic packet to wake up a device with Wake-on-LAN capabilities."
---

> Use this action to send a _magic packet_ to wake up a device with [Wake-on-LAN](https://en.wikipedia.org/wiki/Wake-on-LAN) capabilities, for example to turn on a computer from an automation.

Reviewer: Simplify a bit:

```suggestion
Use this action to send a _magic packet_ to wake up a device by using [Wake-on-LAN](https://en.wikipedia.org/wiki/Wake-on-LAN), for example to turn on a computer from an automation.
```
